cypress/component/unit_test_apollo_state_updates.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Dashboard from '../../src/components/organisms/Dashboard';
+import AddComponentButton from '../../src/components/atoms/AddComponentButton';
+import { dashboardItemsVar } from '../../src/cache/index';
+
+describe('Apollo state management', () => {
+    beforeEach(() => {
+        dashboardItemsVar([]);
```

Comment:
What does invoking this do? Wouldn't it be better to create this variable each time and pass it into the `it` blocks? That way you can ensure you don't encounter any race conditions if e.g. one test runs before the other and mutates the variable?

Reply:
I.e. use `makeVar<DashboardItemInfo[]>([]);` in the `beforeEach` to create a fresh state each test.

Reply:
That is very true, very good point.

Reply:
Hey, me again. Seems like this is easier said than done. We use this variable multiple places in the application (it's just how apollo state manager works), so we cannot make a new variable just for these tests, and we cannot really use makeVar to initialise a fresh state since the variable is imported. So, with my restricted knowledge, I would say that just setting the variable to [] before each test is adequate enough. If this causes any problems down the line, I can owe you a small beverage.

Reply:
Alright. Looking forward to the small beverage (or not).